python/lsst/cp/pipe/cpCombine.py

```diff
@@ -612,6 +615,33 @@ def setFilter(exp, filterLabel):
         """
         pass
 
+    def calibStats(self, exp, calibrationType):
```

Comment:
This could be a static method, I guess.

Reply:
Left alone, as this now uses a config parameter.